src/posts/10year-event/index.md

```diff
@@ -0,0 +1,24 @@
+---
+title: 'Parcels 10 year anniversary event'
+date: '2025-04-01'
+authors:
+  - name: Erik van Sebille
+    github: erikvansebille
+summary: 'On October 1-3 2025, we will celebrate the 10 year anniversary of Parcels in Utrecht, Netherlands, with a workshop-conference-hackathon-party.'
+---
+
+# Parcels 10 year anniversary event 🎉
```

Comment:
This H1 isn't needed as the `title` in the front matter is already rendered on the page